op-conductor/conductor/service.go

```diff
@@ -602,17 +608,17 @@ func (oc *OpConductor) action() {
 		// normal leader, do nothing
 	}
 
-	oc.log.Debug("exiting action with status and error", "status", status, "err", err)
+	oc.log.Debug("exiting action with status and error", "status", status.String(), "err", err)
```

Comment:
Also it seems like `.String()` will be called automatically in log functions, you don't need to explicitly call that